### ISSUE TITLE
Speech feedback in CCleaner does not follow cursor movement when using a portable NVDA copy

### ISSUE THREAD

**Issue by Adriani90** [nvaccess/nvda#7817]
### Steps to reproduce:
1. Create a portable NVDA copy
2. Open CCleaner
3. Move by pressing tab and shift tab through the application
4. Go to a list or tree view in CCleaner and try to read the entries

### Expected behavior:
NVDA should behave like the installed version where speech feedback interrupts when Tab or Shift tab is being pressed. Moreover, in lists NVDA should behave properly that means, entries should be reported when landing on a list and pressing up and down arrow.

### Actual behavior:
Speech feedback does not react when pressing tab or shift tab. Entries in lists and tree views are not being reported.

### System configuration:
NVDA version:
2017.3

NVDA Installed or portable:
Portable

Other information:

Windows version:
Windows 10 Fall creators Update

Name and version of other software in use when reproducing the issue:
CCleaner V5.37.6309

### Other questions:

Does the issue still occur after restarting your PC?
Yes

Have you tried any other versions of NVDA?
Yes, 2017.4 RC3

**Comment by Brian1Gaff** [comment]
No I reported this some time ago. I believe it is because ccleaner is using UIA for these lists in Windows 7 and up, It will still work in XP though. I'm not certain whether nvda could force Ccleaner to reveal this info in the old way to a portable version.I'd defer to somebody better versed in these things.
 However I'd also say that a portable version of nvda in Windows 10 is a lot less functional than it even is in 7, and certainly xp in the old version.
 This seems to be all part of the way Windows only allows trusted software to access some of its parts. I think this is probably all part of the great security push.
 Brian

**Comment by Adriani90** [comment]
Actually this works just fine when running the portable copy as administrator. I guess this is due to system privileges and there is not much NVDA developers can do about it. I am closing as works for me.
